runchecks

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+show_help() {
+  printf "Usage:\n"
```

Comment:
when the final `\n` is needed, why can't you just use `echo` instead? I think that is cleaner. Unless you have a better reason to use `printf` I ask you to use `echo`.

Reply:
`echo` doesn't work so well with `\t`. Sometimes it just prints "\t" in plaintext, so I changed it to `printf`, so it has a more stable behavior.